core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -244,20 +251,24 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
 
       cloudHealthCheckKillThread.get.start()
     }
+  }
 
+  def startUnhealthyStateCheckThread(): Unit = {
     cloudHealthCheckThread = Some(new Thread {
       override def run(): Unit = {
         while (true) {
           Thread.sleep(hc.getConf.healthCheckInterval)
           if (!H2O.CLOUD.healthy()) {
-            Log.err("External H2O cloud not healthy!!")
+            if(hc.getConf.isManualClusterStartUsed){
```

Comment:
nit: missing white space after `if` and before `{`

Reply:
thx!